src/components/grid/row.js

```diff
@@ -22,8 +22,12 @@ const Row = styled.div`
     ? DIMENSIONS.map(d =>
       config(p).breakpoints[d] && config(p).media[d]`
         flex-direction: ${p.reverse.indexOf(d) !== -1 ? `row-reverse` : `row`};
```

Comment:
After testing here, I saw that the flex-wrap is being applied in all cases, including where it shouldn't be reverse. I suggest this small change. What do you think? In the case of passing an array of dimensions to invert, `flex-wrap: wrap-reverse`  and `flex-direction: row-reverse` will be applied only in the dimensions informed
```suggestion
              ${p.reverse.indexOf(d) !== -1
          ? `
            flex-direction: row-reverse;
            flex-wrap: wrap-reverse;
          `
          : 'flex-direction: row;'
        }
      `)
    : `
      flex-direction: row-reverse;
      flex-wrap: wrap-reverse;
    `}
```

Reply:
I'm curious, why do you use the indexOf instead of [`Array.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method to search elements in the array?

Reply:
You're right! The `flex-wrap: wrap-reverse` is being applied even in other dimensions. My bad!